lib/mdi/Install.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python
```

Comment:
There is build support for building with the conventional build system, but not with CMake